src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Process/Detail.html

```diff
@@ -1,8 +1,44 @@
-<div>
+<div data-ng-if="hasResourceHeader">
+    <div class="resource-navigation">
+        <adh-resource-dropdown
+            data-print="true">
+        </adh-resource-dropdown>
+    </div>
+    <header class="resource-header" data-ng-class="{'m-image': data.picture}">
+        <adh-background-image
+            data-no-fallback="true"
+            data-path="{{data.picture}}">
+            <div class="resource-header-main-wrapper">
+                <div class="resource-header-main">
+
+                    <h1 class="resource-header-title">{{ data.title }}</h1>
+
+                    <ul class="resource-header-meta">
+                        <li>
+                            {{ "TR__BY" | translate }} <adh-user-meta data-path="{{data.creator}}" data-ng-if="data.creator"></adh-user-meta>
+                        </li>
+                        <li>
+                            {{ "TR__ON" | translate }} <adh-time data-datetime="data.creationDate" data-format="L"></adh-time>
+                        </li>
+                    </ul>
```

Comment:
I don't think that this meta data is very useful. How about we remove it for now and think about useful meta data separately?